### DISCUSSION TITLE
iOS App Transfer - Will FCM push notification ability be lost?

### DISCUSSION THREAD

**Discussion by dooleyb1** [firebase/firebase-ios-sdk#9610] [Q&A]
I’m considering doing an [iOS App Transfer to another App Store Connect account](https://help.apple.com/app-store-connect/#/deved688524f) however I am concerned about losing push notification ability to iOS users via FCM during the transition. I’ve had a dig around but couldn’t find any documentation directly confirming whether this would / wouldn’t be the case.

The bundleID of the app will not change during the transfer however the [APNs Auth Key](https://firebase.google.com/docs/cloud-messaging/ios/client#upload_your_apns_authentication_key) will have to change. I'm worried this new APNs key will not have the correct permissions / ability to send push notifications to users who may not have updated their app on iOS yet.

This [firebase-ios-sdk GitHub issue](https://github.com/firebase/firebase-ios-sdk/issues/2934) referenced a similar concern but didn't seem to come to any resolution.

Has anyone done this before / does anyone know if we will lose the ability to send push notifications to users who may be on the old build after we have made the app transfer?

Thanks! 😄

**Comment by felix-mr** [comment]
I have same issue. 
Have you solved it?

**Comment by rohit-suthar** [comment]
I have a similar doubt about this. Does anyone have an update?
